src/backend/routers/students.ts

```diff
@@ -17,30 +16,85 @@ export const studentProcedures = {
 
       return result;
     }),
+  //* this function gets students for the current logged in case manager
+  getMyStudents: procedure
+    .input(z.object({ assigned_case_manager_id: z.string().uuid() }))
+    .query(async (req) => {
+      const { assigned_case_manager_id } = req.input;
```

Comment:
going by the function name, I would expect that this would take 0 parameters and instead look at the user ID from the authentication attached to the request--try `req.auth...`

Reply:
Completed. Still have some type-error left behind. Not sure how to go around fixing it.

Reply:
Restarted my vs code and it's working now. I guess I needed to update my vs code.